tests/functional/modules/test_zos_lineinfile_func.py

```diff
@@ -568,7 +587,42 @@
 export PYTHONPATH
 export PKG_CONFIG_PATH
 export PYTHON_HOME
-export _BPXK_AUTOCVT"""),
+export _BPXK_AUTOCVT""",
```

Comment:
This is only a comment, no action needed. We really did not need to extend this sample data to test force but it made it easier for the developer @AndreMarcel99 to see his changes working, in case you are reviewing this I wanted you to understand why the data was extended, although we would ask to remove it, i see no point, its just adding some extra reading time. :)